loops_lab.js

```diff
@@ -0,0 +1,108 @@
+//Write a while loop and a for loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Overall good demonstration of for and while loops. I left some feedback for some problems below. Nothing major but mostly about good practices for loops. Keep up the good work!